src/kv/RocksDBStore.cc

```diff
@@ -366,6 +366,7 @@ void RocksDBStore::get_statistics(Formatter *f) {
   bool status = db->GetProperty("rocksdb.stats", &stats);
   if (status) {
     f->open_object_section("bluestore rocksdb statistics");
+    f->write_raw_data("\n We usually observe an overhead of 5%-10% for this command. Be carefully before using\n\n");
```

Comment:
I'm afraid that's not enough - performance impact starts when you create statistics object and assign it to DB instance in do_open method. Hence performance is affected permanently, not after command execution.